source/avansert-flyt.rst

```diff
@@ -0,0 +1,24 @@
+Avansert flyt
+***************
+
+I tillegg til å kunne tilby :ref:`signering-i-direkteflyt` og :ref:`signering-i-portalflyt`, så har Posten signering lagt opp til å støtte mer avanserte flyter.
+
+Kjedet signatur
+=================
+
+Avsender kan spesifisere rekkefølgen et signeringsoppdrag skal bli tilgjengeliggjort for undertegnerne for oppdrag som :ref:`signeres i portalflyt <signering-i-portalflyt>`. Når alle undertegnerne i en gruppe har signert, vil oppdraget bli tilgjengelig for neste gruppe undertegnere. En gruppe undertegnere er alle som har samme rekkefølge (:code:`order`) i APIet og kan bestå av én eller flere undertegnere som skal signere i parallell.
```

Comment:
Stemmer egentlig dette - at det kun er portalflyten som man kan spesifisere rekkefølgen på? Trodde det bare gjaldt API-et, og egentlig at det er direkte-signering?
--
Kanskje litt tung setning på starten der.
Forslag: "For oppdrag som :ref:`signeres i portalflyt <signering-i-portalflyt>`, kan avsender spesifisere en rekkefølge som undertegnerne skal signere i." 

eller bare

"Avsendere som bruker portalflyt kan spesifisere en signeringsrekkefølge for undertegnerne".

Reply:
Det er portal-oppdrag det i hovedsak er tenkt for. Lurer på om det ble lagt til en mulighet for å å kunne ha rekkefølge på et direkte-oppdrag, men da blir den første undertegneren i direkteflyt, mens resten blir portalflyt.

Reply:
Aha! Da er det jo kanskje litt rart vi ikke tilbyr denne muligheten i avsenderportalen også! (hvis det gjelder portalflyt)

Reply:
Teksten er endret, har lagt til at vi skal sjekke order for direkteflyt, men det tenker jeg vi kan ta senere, ettersom vi vet det som står nå er korrekt. Det er ev. et spørsmål om vi skal legge til et eksempel for direkteflyt, men det tar vi siden :)